scripts/wdl/case_gatk_acnv_workflow_template.json

```diff
@@ -25,4 +25,15 @@
   "case_gatk_acnv_workflow.jni_lib": "FILL_ME_IN", /* Broad internal: /broad/software/free/Linux/redhat_6_x86_64/pkgs/hdfview_2.9/HDFView/lib/linux/  (assumes ``use .hdfview-2.9``) */
   "case_gatk_acnv_workflow.plots_dir": "FILL_ME_IN", /* path to store produced plots for CopyRatio and ACNV */
   "case_gatk_acnv_workflow.enable_gc_correction": "true" /* true/false option change to false to bypass gc bias correction step */
+  "case_gatk_acnv_workflow.seg_param_alpha": "FILL_ME_IN", /* Segmentor parameter, default=0.01 */
```

Comment:
We have been spelling it `segmenter`, which you should do for consistency, although both might be correct.